base/domain-xho.yml

```diff
@@ -0,0 +1,377 @@
+  - utter_ask_medical_condition_hypertension
+  - utter_ask_medical_condition_cardio
+  - utter_start_health_check
+  - utter_ask_symptoms_fever
+  - utter_ask_symptoms_cough
+  - utter_ask_symptoms_sore_throat
+  - utter_ask_symptoms_difficulty_breathing
+  - utter_ask_symptoms_taste_smell
+  - utter_ask_exposure
+  - utter_ask_tracing
+  - utter_submit
+  - utter_risk_low
+  - utter_risk_moderate
+  - utter_risk_high
+
+slots:
+  terms:
+    type: categorical
+    values:
+      - yes
+  age:
+    type: unfeaturized
+    auto_fill: false
+  gender:
+    type: unfeaturized
+    auto_fill: false
+  province:
+    type: unfeaturized
+    auto_fill: false
+  location:
+    type: unfeaturized
+    auto_fill: false
+  location_confirm:
+    type: unfeaturized
+    auto_fill: false
+  location_coords:
+    type: unfeaturized
+    auto_fill: false
+  city_location_coords:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_obesity:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_diabetes:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_hypertension:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_cardio:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_fever:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_cough:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_sore_throat:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_difficulty_breathing:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_taste_smell:
+    type: unfeaturized
+    auto_fill: false
+  exposure:
+    type: unfeaturized
+    auto_fill: false
+  tracing:
+    type: unfeaturized
+    auto_fill: false
+  requested_slot:
+    type: unfeaturized
+
+forms:
+  - healthcheck_form
+  - healthcheck_terms_form
+  - healthcheck_profile_form
+
+responses:
+  utter_welcome:
+      - text: |
+          *HealthCheck* sisixhobo sakho sokuhlola umngcipheko ngeveki. Nceda usincede ngokuphendula imibuzo embalwa malunga nawe nempilo yakho. Uza kufumana ingcebiso malunga nokuba manyathelo mani onokuwathatha ngokusekwe kwizikhokelo zezwelonke kwaye  le datha oyifakileyo iya kusinceda kwimodeli yokuqikelela kunye nokucwangcisa impendulo yethu yezwelonke nge-COVID-19. Enkosi ngokuza ngaphambili nangegalelo kwimpilo yabo bonke abahlali KUNYE nasekunqandeni ukusasazeka kwale ntsholongwane  🦠
+
+  utter_welcome_back:
+      - text: |
+          Welcome back to *HealthCheck*, your weekly risk assessment tool. Thank you for coming forward and for contributing to the health of all citizens AND stopping the spread of the virus 🦠
+
+  utter_ask_terms:
+      - text: |
+          Phendula ngegama elinye kula ANGONOBUMBA ukuze uqhubekeke
+
+          YAMKELA ✅ - _ukwamkela imigaqo nemiqathango_
+          OKUNINZI 📄 - -ukufunda imigaqo nemiqathango_
+
+          (okanye phendula iMENYU 📌 _ukubuyela kwimenyu yakuqala_)
```

Comment:
Keywords need to be bolded

Reply:
We'll need to add this keyword as one of the exit keywords in the NLU training data